app/views/questionnaire.py

```diff
@@ -457,6 +538,8 @@ def _get_context(block, current_location, answer_store):
 
     error_messages = SchemaHelper.get_messages(g.schema_json)
     form, template_params = get_form_for_location(block, current_location, answer_store, error_messages)
+    form.validate_on_submit()
```

Comment:
What is this for?

Reply:
Um... Yeah, that's random. I had a lot of fun with rebase earlier in the week so I'm going to blame it on that.